script/client/conf/registry.conf

```diff
@@ -8,39 +8,47 @@ registry {
     application = "seata-server"
     serverAddr = "127.0.0.1:8848"
     group = "SEATA_GROUP"
+    cluster = "default"
```

Comment:
I think the file needs to reset. The service name of TC is get from the configuration center instead of being set to registry.conf.

Reply:
When do not use configuration center, will need it.

Reply:
done